lib/exceptions/application_validator.js

```diff
@@ -87,6 +87,9 @@ function checkSpecificOptions(option, value) {
     case 'prodDatabaseType':
       checkProdDatabaseTypeValue(value);
       break;
+    case 'creationTimestamp':
```

Comment:
that implies that the creationTimestamp is an app option, I don't think it is, is it?

Reply:
What do you mean about app option? jdl option?

Reply:
generator-jhipster should forward it if passed as an arg.

Reply:
This is not needed.

Reply:
in the .yo-rc file too?

Reply:
It writes to .yo-rc file for a rebuild with `jhipster --with-entities`

Reply:
okay then, it's an acceptable coupling for me